src/panels/config/automation/ha-automation-editor.ts

```diff
@@ -113,6 +117,37 @@ export class HaAutomationEditor extends KeyboardShortcutMixin(LitElement) {
             ><ha-svg-icon path=${mdiDotsVertical}></ha-svg-icon>
           </mwc-icon-button>
 
+          <mwc-list-item
+            aria-label=${this.hass.localize(
+              "ui.panel.config.automation.editor.edit_ui"
+            )}
+            graphic="icon"
+          >
+            ${this.hass.localize("ui.panel.config.automation.editor.edit_ui")}
+            ${this._mode === "gui"
+              ? html`<ha-svg-icon
+                  slot="graphic"
+                  .path=${mdiCheck}
+                ></ha-svg-icon> `
+              : ``}
```

Comment:
use `<ha-button-menu activatable>` and `<mwc-list-item selected activated>...</mwc-list-item>`

Reply:
Could be besides the check

Reply:
Looks great!